Cesium.Compiler/Main.cs

```diff
@@ -3,7 +3,9 @@
 using Cesium.Compiler;
 using CommandLine;
 
-return await Parser.Default.ParseArguments<Arguments>(args).MapResult(async args =>
+var parserResult = new Parser(x => x.HelpWriter = null).ParseArguments<Arguments>(args);
```

Comment:
Why do you set `TextWriter = null` here? I don't think it is required.

Reply:
It prints 2 times in console if not set to null